### ISSUE TITLE
Redesign Footer for Improved User Experience

### ISSUE THREAD

**Issue by kekubhai** [Rakshit-gen/Slanine#58]
The current footer section is taking up excessive space on the page and is not visually appealing. The design is cluttered and lacks clear navigation, potentially hindering user interaction. To enhance the user experience, we propose redesigning the footer to be more compact, visually appealing, and user-friendly. This involves reducing the amount of content, reorganizing elements for better clarity, and incorporating more intuitive navigation options. By implementing these changes, we can create a footer that is both functional and aesthetically pleasing, contributing to a more positive overall user experience.

**Comment by Rakshit-gen** [comment]
Sure @kekubhai assigning the issue to you,
Kindly provide an image of the design you would like to add then send a PR

**Comment by kekubhai** [comment]
I was thinking of adding an bg to the footer with this this : 
![Screenshot 2024-10-14 213436](https://github.com/user-attachments/assets/be0706d1-3534-4841-b229-369878635f6a)
I guess It would go great, tell me what you like or adding a starry theme footer bg will be good ?

**Comment by Rakshit-gen** [comment]
Ummm, not this one, can you find something new and unique. It will make the website heavier and slower.

**Comment by kekubhai** [comment]
Yeah you are right right . I guess I will find some great static designs and when I am done I will show you how it looks, ASAP

**Comment by kekubhai** [comment]
hey there , Check this one : 
![image](https://github.com/user-attachments/assets/163f3995-bd9f-46d8-89c5-6ab0e389e272)
I guess this goes good with the UI and the overall background. Should I add a PR ?

**Comment by Rakshit-gen** [comment]
Okay this one looks better @kekubhai.
Update this one

**Comment by kekubhai** [comment]
Good to hear that you liked it.  Raising a PR . Thank you.
